src/lib/aragon-core.js

```diff
@@ -6,9 +6,12 @@ const networks = require('./networks')
 
 IPFS_TIMEOUT = 600000 // 10min
 
-async function getLatestFromRepo (repo, network) {
+async function getLatestFromRepo (repo) {
+  const web3 = new Web3(provider())
+  const network = await web3.eth.net.getNetworkType()
+
   const networkConfig = networks[network]
-  if (!networkConfig) {
+  if (!networks[network] && network !== 'local') {
```

Comment:
```suggestion
  if (! networkConfig && network !== 'local') {
```

Although I'm not sure what would happen on `local`; I don't know if we publish aragon/aragon on aragen (cc @0xGabi)

Reply:
No, we didn't publish `aragon/aragon` on current version. I just open an issue 👍